cibuildwheel/windows.py

```diff
@@ -132,6 +133,14 @@ def shell(args, env=None, cwd=None):
         simple_shell(['pip', '--version'], env=env)
         simple_shell(['pip', 'install', '--upgrade', 'setuptools', 'wheel'], env=env)
 
+        # setuptools does not setup the compiler correctly on GH Actions for python 3.5
+        # c.f. https://github.com/pypa/setuptools/issues/1851#issuecomment-537714885
+        # Let's patch setuptools with distutils discovery mechanism from 3.8.0
+        shutil.copyfile(
+            os.path.join(os.path.dirname(__file__), 'resources', 'msvc.py'),
+            os.path.join(config_python_path, 'Lib', 'site-packages', 'setuptools', 'msvc.py')
+        )
```

Comment:
This is quite a funny hack! :) I guess it'll only last until this PR is merged/released? https://github.com/pypa/setuptools/pull/1904

Reply:
yes

Reply:
This is a big hack while waiting for upstream PR, I'm not keen on getting the hack baked into cibuildwheel.

Reply:
This hack is not needed anymore, setuptools has been released with the patch.